cypress/integration/rendering/sankey.spec.js

```diff
@@ -0,0 +1,15 @@
+import { imgSnapshotTest } from '../../helpers/util.js';
+
+describe('Sankey Diagram', () => {
+  it('should render a simple example', () => {
+    imgSnapshotTest(
+      `
+      sankey-beta
+      
+      a,b,10
+      `,
+      {}
+    );
+    cy.get('svg');
```

Comment:
There is no need for this, it would be called inside `imgSnapshotTest` function anyway.

Reply:
Done